lib/travis/build/script/go.rb

```diff
@@ -149,7 +150,7 @@ def comparable_go_version
           end
 
           def go_get_cmd
-            (go_version == 'go1' || comparable_go_version < Gem::Version.new('1.2')) ? 'go get' : 'go get -t'
+            (go_version == 'go1' || (go_version != 'tip' && comparable_go_version < Gem::Version.new('1.2'))) ? 'go get' : 'go get -t'
```

Comment:
The logic is getting a bit too complicated for a ternary operator. Please rework this into a more simple `if` statement.